azure-ts-msi-keyvault-rbac/webapp/Program.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
```

Comment:
do we have copyright in our example files?

Reply:
No `*.cs` files do, so far. Some `*.ts` have them, some not.